ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -124,11 +152,26 @@ open class PillButton: UIButton {
 
     private func updateAppearance() {
         if let window = window {
-            backgroundColor = isSelected ? style.selectedBackgroundColor(for: window) : style.backgroundColor(for: window)
+            if (isSelected && !isEnabled) {
```

Comment:
can reorganized this better?
if selected {
  if isenabled
{
  background =
  titla = 
 } else {
 background = 
 title = 
 } 
}
else {
}

Reply:
updated

---
In reply to: [459631312](https://github.com/microsoft/fluentui-apple/pull/133#discussion_r459631312) [](ancestors = 459631312)